installer/Windows Server 2019/installer/install.cmd

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
why "Windows Server 2019" folder ? why not just "windows" ?

Reply:
modified now to the family of os names!!

Reply:
Renamed to linux,macos and windows !!